src/components/sytles/navbar.css

```diff
@@ -0,0 +1,64 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  width: 87.5%;
+  padding: 1.438% 6.188% 1.688% 0.8%;
+  background-color: #fff;
+  box-shadow: 5px 5px 50px rgba(52, 58, 62, 0.7);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: auto;
+}
+
+a {
+  text-decoration: none;
+  font-size: 1.4em;
+}
+
+#logo {
+  width: 20%;
+  margin: 0.375% 0 0.125% 0;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
- [ ] (Required) I noticed you are repeating this rule a lot of times in this file. I would like to advise that you either turn it to a class selector (`.montserrat`) and use it where necessary or you make it a CSS variable and use it in the file.

Reply:
Fixed! 😄